src/ProblemDetailsResponseFactory.php

```diff
@@ -213,7 +215,8 @@ public function __construct(
         };
         $this->isDebug = $isDebug;
         if (! $jsonFlags) {
-            $jsonFlags = JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE | JSON_PRESERVE_ZERO_FRACTION;
+            $jsonFlags = JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE | JSON_PRESERVE_ZERO_FRACTION
+                | JSON_PARTIAL_OUTPUT_ON_ERROR;
```

Comment:
If you're going to break this onto another line, do a line per `|`, please.

Reply:
Done